LICENSE

```diff
@@ -1,4 +1,4 @@
-This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+This work by Kyle R. Conway (https://kylerconway.com) is licensed under the Creative Commons Attribution-ShareAlike 4.0
```

Comment:
Getting back to the details of the pull-request, could you update the pull-request by restoring the LICENSE file to the original state?  I would do it for you, but that might look a little odd in the history.

You can either do it manually by creating a new patch or you can use git's built in "revert" functionality to revert the LICENSE change (e.g. commit f289cdb6f4e2b21a25df48006896dfa1eed538aa).  The links below provide some more information on how to do a "git revert", but if you have any questions please let me know.

Thanks!

* https://www.atlassian.com/git/tutorials/undoing-changes#git-revert
* https://git-scm.com/docs/git-revert